examples/specs/window_sum_waterfall_chart.vl.json

```diff
@@ -96,8 +111,8 @@
         "color": "#404040"
       },
       "encoding": {
-        "y": {"field": "sum_inc", "type": "quantitative"},
```

Comment:
Not sure why do you have to remove all whitespaces after comma.  It's actually a good thing to have them.

Reply:
I'm pretty sure that all I did was changing tick to rule, adding window, and adding offsets...

Reply:
Hmm, some autoformatter must be acting weird.

Reply:
I figured it out. I was trying to use the old editor, and it gave me that format. Now, I use the new editor, and it works fine.